app/controllers/order_products_controller.rb

```diff
@@ -0,0 +1,94 @@
+class OrderProductsController < ApplicationController
+  before_action :find_order_product, except: [:index, :new, :create]
+
+  def index
+    @order_products = OrderProduct.all
+  end
+
+  def show
+  end
+
+  def new
+    @order_product = OrderProduct.new
+  end
+
+  def create
+    find_cart
+
+    already_in = false
+
+    @order.order_products.each do |order_product|
+      if order_product.product_id.to_i == params[:order_products][:id].to_i
+        already_in = true
+        @order_product = OrderProduct.find(order_product.id)
+        @order_product.save
```

Comment:
You find an order_product and then save it to the database without changing anything here.